str0m/src/media/sender.rs

```diff
@@ -15,8 +16,15 @@ pub struct SenderSource {
     rid: Option<Rid>,
     next_seq_no: SeqNo,
     last_used: Instant,
-    bytes_tx: u64,
-    bytes_tx_resent: u64,
+    // bytes includes resent
+    bytes: u64,
+    bytes_resent: u64,
+    // packets includes resent
```

Comment:
That `bytes` and `packets` also holds the resent is according to spec I assume?

Can we clarify the comment that this is according to spec? Maybe even link to the spec?

Reply:
correct, ok

Reply:
I add the links to the stats events, not the sender/receiver because that is where the "public api" of the stats is